packages/sdk/contractkit/src/wrappers/OdisPayments.ts

```diff
@@ -0,0 +1,26 @@
+import { CeloTransactionObject } from '@celo/connect'
+import { OdisPayments } from '../generated/OdisPayments'
+import { BaseWrapper, proxyCall, proxySend } from './BaseWrapper'
+
+type Address = string
+
+export class OdisPaymentsWrapper extends BaseWrapper<OdisPayments> {
+  /**
+   * @notice Fetches total amount sent (all-time) for given account to odisPayments
+   * @param account The account to fetch all-time quota for
```

Comment:
nit: replace 'quota' with 'funds' or 'payments'. we decided on using quota to refer to the number of queries available to the user and payments to refer to the money they've paid towards obtaining quota, to make things a little less confusing

Reply:
yes